aten/src/ATen/native/cuda/Distributions.cu

```diff
@@ -730,7 +730,14 @@ Tensor& normal_out_cuda(Tensor& output, const Tensor& mean, const Tensor& std, G
   // The previous function here was addcmul_out(output, mean, output, std, 1);
   // The third argument is not a constant reference and hence the samples in output are overwritten.
   // Consequently, the computation performed is mean + mean * std instead of mean + output * std
-  output.mul_(std).add_(mean);
+  if (mean.numel() == std.numel() && !at::is_expandable_to(std.sizes(), mean.sizes())) {
+    TORCH_WARN_ONCE("Reshape std to the same shape of mean due to not broadcastable");
```

Comment:
this isn't really a good description of the situation.  It's not clear from the user's perspective how this information is actionable.

How about something like:
"std and mean have the same number of elements, but are not broadcastable.  This was previously a supported mode of operation, but is now deprecated and the support will be removed in a later release.  Note that the current implementation reshapes std to the shape of mean, which may be incur data copies.  Please ensure that std and mean are broadcastable to avoid these issues."

Reply:
I also don't think `is_expandable_to` is the correct function to call -- that  name suggests directionality, but you really just want to know if the two tensors are broadcastable.

Reply:
please add a test of this case so we can be sure we don't make this mistake.